### DISCUSSION TITLE
Sort aliases by recent behavior

### DISCUSSION THREAD

**Discussion by g000m** [anonaddy/anonaddy#497] [Feature Requests]
This is similar to #407  ; I'd like to be able to sort aliases by how recently they received an email. I think just storing a timestamp would suffice.

**Comment by luigicalamar** [comment]
Yes - just add two columns to the alias - last forwarded date and created date

On the alias page:  (https://app.addy.io/aliases) - would be great if two sortable date columns were added:

1) Last date message forwarded

2) Date alias created

For 1) - I'd like to be able to see how long since a message was forwarded with that Alias... would be a nice way to 'clean up' aliases no longer needed

for 2) - the delta between created date and last forwarded date - would be useful to determine how long since the alias was created and last received a message with that alias

**Comment by willbrowningme** [comment]
This is now possible.
